docs/src/main/sphinx/installation/cli.rst

```diff
@@ -46,17 +46,30 @@ The recommended TLS implementation is to use a globally trusted certificate. In
 this case, no other options are necessary, since the JVM running the CLI
 recognizes these certificates.
 
+Use ``--help`` to see information about specifying the keystore, truststore, and
+other authentication details as required.
+
 Authentication
 --------------
 
-You can override your username with the ``--user`` option. It defaults to your
-operating system username. If your Trino server requires password
-authentication, use the ``--password`` option to have the CLI prompt for a
-password. You can set the ``TRINO_PASSWORD`` environment variable with the
-password value to avoid the prompt.
+The Trino CLI supports many :doc:`/security/authentication-types` detailed in
+the following sections:
 
-Use ``--help`` to see information about specifying the keystore, truststore, and
-other authentication details as required. If using Kerberos, see :doc:`/security/cli`.
+Username and password authentication
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Username and password authentication is typically configured in a cluster with
+the ``PASSWORD`` authentication type. It defaults to your operating system
+username. If your cluster uses a different username, you can override your
+username with the ``--user`` option.
```

Comment:
This part about username and it's default value isn't applicable only when password auth is enabled. So it should precede this section.

Reply:
Good point .. how about we leave this as is in this PR.. but move it into the table of options we are working on  in a follow up PR @hashhar  ..

Reply:
Fine with me, can you link me to the other PR so that I remember to make sure this is applied?

Reply:
@hashhar https://github.com/trinodb/trino/pull/11187/files